src/ExpressionLanguage/ExpressionFunction/GraphQL/Resolver.php

```diff
@@ -12,8 +13,12 @@ public function __construct($name = 'resolver')
     {
         parent::__construct(
             $name,
-            function ($alias, $args = '[]') {
-                return \sprintf('$globalVariable->get(\'resolverResolver\')->resolve([%s, %s])', $alias, $args);
+            function (string $alias, string $args = '[]'): string {
+                return "\$globalVariable->get('resolverResolver')->resolve([$alias, $args])";
+            },
+            // This expression function is not designed to be used by it's evaluator
+            function (): void {
```

Comment:
you can use `new EvaluatorIsNotAllowedException($name)` since Exception is callable

Reply:
Done!